File: aggregator/java-spring-kafka-streams-aggregator/src/main/java/com/uuhnaut69/kafkastream/event/Dish.java
package com.uuhnaut69.kafkastream.event;

import com.fasterxml.jackson.databind.PropertyNamingStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
public class Dish {

  private String id;

  private String name;

  private BigDecimal price;

  private Long createdDate;

  private String restaurantId;
}

File: aggregator/kotlin-spring-cloud-streams-aggregator/src/main/kotlin/com/uuhnaut69/kcloudstreams/event/Dish.kt
package com.uuhnaut69.kcloudstreams.event

import com.fasterxml.jackson.databind.PropertyNamingStrategy
import com.fasterxml.jackson.databind.annotation.JsonNaming
import java.math.BigDecimal

@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy::class)
data class Dish(
    val id: String,
    val name: String,
    val price: BigDecimal,
    val createdDate: Long,
    val restaurantId: String
)
